src/Articles.jsx

```diff
@@ -24,7 +24,7 @@ if (isLoading) {
     return <div>Won't be long, just loading...</div>
 }
 
-if (error)
+if (isError)
 return <div>Error: {error.message} </div>
```

Comment:
When the request fails you get the following runtime error 
<img width="506" alt="image" src="https://github.com/emmajohnston93/fe-nc-news/assets/105444123/dfbd4fa1-2d39-4bbc-9369-2c6a602c0205">
- I believe there may still be some references to the old variable name in this file and in `SingleArticle`